lib/l10n/app_de.arb

```diff
@@ -318,5 +318,12 @@
   "cardPendingRideRejectDialogMessage": "Bist du sicher, dass du diese Mitfahrt ablehnen möchtest?",
   "cardPendingRideRejectDialogSuccessSnackBar": "Mitfahrt abgelehnt",
 
+  "pageChatEmptyTitle": "Nichts zu sehen",
+  "pageChatEmptyMessage": "Starte jetzt eine Konversation :)",
+  "pageChatNoChatMessage": "Du kannst mit deiner*m Fahrer*in chatten,‚ sobald deine Fahrt angenommen wurde.",
```

Comment:
here is a doubled comma, but for pageDriveChatEmptyMessage it is missing completely...